content/espanol/sql_workshop/basics_activity/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Tutorial 1: Aprende los pasos básicos de un cadete"
+description: "aprende qué es una base de datos y SQL"
+date: 2020-07-26
+prereq: "none"
+difficulty: "Principiante"
+icon: ""
+draft: false
+---
+<p style="text-align: center;"><iframe width="560" height="315" src="https://www.youtube.com/embed/l5BDCMyFHVI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>
```

Comment:
Delete video

Reply:
Removed the youtube video of all the activities